include/sdf/Sphere.hh

```diff
@@ -65,6 +68,12 @@ namespace sdf
     /// not been called.
     public: sdf::ElementPtr Element() const;
 
+    /// \brief Calculate and return the Mass Matrix values for the Sphere
+    /// \param[in] density Density of the sphere in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the sphere in kg/m^3
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)